README.md

```diff
@@ -103,29 +102,76 @@ cd GameplayFootball
 
 % Switch to windows branch
 git switch windows
+```
 
+Now we have 2 options:
 
-% Copy the contents of `data` directory into `build\Debug` or (and) `build\Release`
-xcopy /e /i data build\Debug
-xcopy /e /i data build\Release
-```
-Go to `build` directory and generate `cmake` files. Make sure that you correctly set the directory for `vcpkg` (in our case it is installed into `C:\dev\vcpkg`):
-```bat
-cd build
+### 1. Visual Studio 2019 - Support Debugging and Profiling game
 
-cmake .. -DCMAKE_GENERATOR_PLATFORM=Win32 -DCMAKE_TOOLCHAIN_FILE=C:/dev/vcpkg/scripts/buildsystems/vcpkg.cmake -DCMAKE_WINDOWS_EXPORT_ALL_SYMBOLS=TRUE  
-```
-To build `Release` version:
+Download and install:
+- [Visual Studio 2019 with Desktop development with C++](https://visualstudio.microsoft.com/downloads/),
+ 
+![Desktop C++](https://docs.microsoft.com/en-us/cpp/build/media/vscpp-concierge-choose-workload.gif?view=msvc-150)
+- [C++ CMake tools for Windows](https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-160).
+
+![CMake tool](https://docs.microsoft.com/en-us/cpp/build/media/cmake-install-2019.png?view=msvc-160)
+
+Build project
+- Start `Visual Studio 2019`, then click `File > Open > Folder...`
+- Navigate to the `GameplayFootball` project folder, and click `Select Folder`
+- Select `x86-Debug` or `x86-ReleaseDebugSymbol` and `x86-Release` mode in drop down menu `Manage Configurations...`
+
+![Manage Configurations...](https://docs.microsoft.com/en-us/visualstudio/ide/media/understanding-build-configurations/active-config.png?view=vs-2019)
+- Click `gameplayfootball.exe` in menu `Select startup item...` to start build game
+
+![Select startup item...](https://docs.microsoft.com/en-us/cpp/build/media/cmake-run-button.png?view=msvc-160)
+- After build success, game will start but it close immediately, because it missing data. Now go to step copy data to game
+
+To copy data to game, switch back to Command Prompt window
```

Comment:
I like the way you extensively comment on setting up the project in Visual Studio and VS Code. I'm even thinking of moving it to a separate file (like `development.md`) and adding similar instructions for other platforms/tools once I merge branches. But let's leave it as it is for now.

Reply:
Hi @vi3itor, some "new" VS 2019 features will make old user or newbie (who never use VS and use old document for reference) is hard to read tutorial, so I add image for easier to read it